app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/SearchFragment.java

```diff
@@ -75,8 +76,10 @@ private void filter(String text) {
         ArrayList<Memory> filteredlist = new ArrayList<>();
 
         for (Memory item : memories) {
+            Log.wtf("bla", "location:" + item.getCountryName(getContext()));
```

Comment:
This line is no longer necessary. Please remove it.

Reply:
We don't want to spam the logs unnecessarily.

Reply:
exactly

Reply:
precise